data/mods/DinoMod/monsters/dinosaur.json

```
@@ -1251,5 +1251,54 @@
     "anger_triggers": [ "STALK", "PLAYER_WEAK", "HURT" ],
     "placate_triggers": [ "MEAT" ],
     "categories": [ "DINOSAUR" ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "mon_qianzhousaurus",
+    "name": { "str_sp": "Qianzhousaurus" },
+    "species": "DINOSAUR",
+    "default_faction": "qianzhousaurus",
+    "symbol": "d",
+    "color": "white_brown",
+    "volume": "5000 L",
```

```suggestion
    "volume": "550 L",
```
Match zombie and not disappear body